packages/glob/src/internal-search-state.ts

```diff
@@ -0,0 +1,9 @@
+export class SearchState {
```

Comment:
Can this just be an interface?

Reply:
its internal, doesnt really matter either way. i thought the constructor was slightly nicer to work with

Reply:
let me know if i'm overlooking something else. seems like arbitrary preference either way

Reply:
It's preference, I typically just use interfaces when I only have data and no functions, as then it's just a type definition on an object.